modules/meta/src/test/scala/io/janstenpickle/trace4cats/meta/TracedSpanExporterSpec.scala

```diff
@@ -101,7 +101,11 @@ class TracedSpanExporterSpec
           _ <- Resource.eval(tracedExporter.exportBatch(batch))
 
           allSpans <- Resource.eval(
-            Stream.fromQueueUnterminated(queue).take(batch.spans.size + expectedMetaSpans).compile.to(Chunk)
+            Stream
+              .fromQueueUnterminated(queue)
+              .take(batch.spans.size + expectedMetaSpans)
+              .compile
+              .to(Chunk): IO[Chunk[CompletedSpan]]
```

Comment:
For some weird reason, the Scala 3 compiler can't figure that out by itself, gives up on `collector.Out`.